plugins/httpapi/dcnm.py

```diff
@@ -271,21 +273,26 @@ def send_request(self, method, path, json=None):
 
         self.check_url_connection()
 
+        msg = '". Please verify your login credentials, access permissions and fabric details and try again '
+
         try:
             # Perform some very basic path input validation.
             path = str(path)
             if path[0] != "/":
                 msg = "Value of <path> does not appear to be formated properly"
                 raise ConnectionError(self._return_info(None, method, path, msg))
             response, rdata = self.connection.send(
-                path, json, method=method, headers=self.headers, force_basic_auth=True
+                path, json, self.retrycnt, method=method, headers=self.headers, force_basic_auth=True
             )
             return self._verify_response(response, method, path, rdata)
         except Exception as e:
-            eargs = e.args[0]
+            if e.args:
```

Comment:
Please add a few comments to indicate why this `e.args` check is needed

Reply:
Changes done as suggested